Kerdy/Kerdy/Filter/Views/ResetBtnView.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  ResetBtnView.swift
+//  Kerdy
+//
+//  Created by 이동현 on 11/3/23.
+//
+
+import UIKit
+import SnapKit
+
+class ResetBtnView: UIView {
+
+    private lazy var imageView: UIImageView = {
+        let imageView = UIImageView()
+        imageView.image = UIImage(named: "ic_reset")
+        return imageView
+    }()
+
+    private lazy var titleLabel: UILabel = {
+        let label = UILabel()
+        label.text = "전체 초기화"
+        label.font = .nanumSquare(to: .bold, size: 12)
+        label.textColor = .kerdyGray02
+        return label
+    }()
+
+    private lazy var resetBtn: UIButton = {
+        let button = UIButton()
+        button.setTitle(nil, for: .normal)
+        button.backgroundColor = .clear
+        return button
+    }()
+
+    init() {
+        super.init(frame: .zero)
+        setLayout()
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    private func setLayout() {
+        addSubviews(imageView)
+        addSubviews(titleLabel)
+        addSubviews(resetBtn)
```

Comment:
addSubviews를 사용하실거라면 아래처럼 구현하실 수 있습니다 : ) 그
```suggestion
        addSubviews(imageView, titleLabel, resetBtn)
      
```